### DISCUSSION TITLE
Accidently pushed my .env file to GitHub

### DISCUSSION THREAD

**Discussion by jimguistical** [nss-evening-cohort-14/evening-client-side#191] [Help Me!] [answered]
Jim Conner
Have you searched the other help tickets? Yes
Describe your issue: I accidently pushed my .env file and had somehow not added the file path to my .gitignore file.  I am thinking I must have edited it when I pulled over the webpack.  Sigh.

**_IF THIS HAPPENS TO YOU please don't panic like I did_** -- per Dr. T, just delete the firebase app and redo it.  I followed the  GitHub doc linked below, but it was not necessary for my situation.  If you have deployed your web app with authenticated users, reach out to the Instruction Team because this would be a separate, more complicated issue.

What have you tried? Tell us everything you have tried.
I followed the github doc below, but it wasn't necessary in my case:
https://gist.github.com/liitfr/39c4b0df7be7bfdde1273125f4169dbb

https://www.reddit.com/r/github/comments/jxsuqm/accidentally_pushed_my_env_file/
https://stackoverflow.com/questions/56336890/i-accidentally-uploaded-env-file-with-google-key-in-it-to-github-but-edited-the

Commit link for reference:
https://github.com/jim-conner/personal-bio-site/commit/505e696a1dcea1e2b16a03c4e915ca2cd5f24eaf

**Comment by drteresavasquez** [comment, marked as answer]
As your answer suggests don't panic! 🤣
